src/Heading.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div>
+      <header className="App-header">CYF Hotel</header>
+      <img
+        className="hotel-logo"
+        src="https://cdn.dribbble.com/users/4031439/screenshots/10789004/media/4b07136e303947c2f2075b5b1995cf20.jpg?compress=1&resize=400x300&vertical=top"
+        alt=""
```

Comment:
It's good to get in to the habit of filling in `alt` for users on screen readers